tools/sigh.js

```diff
@@ -273,13 +273,17 @@ function test(args) {
     let runner = `
       import mocha from '${mochaInstanceFile}';
       ${chainImports.join('\n    ')}
-      mocha
-        .grep(${JSON.stringify(options.grep || '')})
-        .run(function(failures) {
-          process.on("exit", function() {
-            process.exit(failures > 0 ? 1 : 0);
+      import debugChannel from '${fixPathForWindows(path.resolve(__dirname, '../debug/connection-channel.js'))}';
+      (async () => {
+        ${options.inspect ? 'console.log("Waiting for Arcs Devtools Extension"); await debugChannel.get().ready;' : ''}
```

Comment:
Does this mean --inspect only works when the extension connects to the websocket?

Reply:
Yes. I added a separate flag --explore to wait for the Arcs Explorer, i.e. extension.